application/src/main/java/org/togetherjava/tjbot/commands/system/CommandSystem.java

```diff
@@ -99,19 +101,21 @@ public CommandSystem(@NotNull JDA jda, @NotNull Database database) {
 
     @Override
     public void onReady(@NotNull ReadyEvent event) {
+        JDA jda = event.getJDA();
+
         // Register reload on all guilds
         logger.debug("JDA is ready, registering reload command");
-        event.getJDA()
-            .getGuildCache()
+        jda.getGuildCache()
             .forEach(guild -> COMMAND_SERVICE.execute(() -> registerReloadCommand(guild)));
         // NOTE We do not have to wait for reload to complete for the command system to be ready
         // itself
         logger.debug("Command system is now ready");
 
         // Propagate the onReady event to all commands
         // NOTE 'registerReloadCommands' will not be finished running, this does not wait for it
-        nameToSlashCommands.values()
-            .forEach(command -> COMMAND_SERVICE.execute(() -> command.onReady(event)));
+        nameToSlashCommands.values().forEach(command -> {
+            COMMAND_SERVICE.execute(() -> command.onReady(event));
+        });
```

Comment:
```suggestion
        nameToSlashCommands.values()
            .forEach(command -> COMMAND_SERVICE.execute(() -> command.onReady(event)));
```

Just a guess to what sonarcloud is talking about